### PR TITLE
[Sival]  update testplans

### PR DESCRIPTION
**Pull request by engdoreis**
Map new tests as discussed on the last SiVal meeting.

### CODE REVIEW

**Review by jwnrt — APPROVED** [review]
LGTM, thanks. I don't know if the OTTF tests need a `si_stage` but I guess it doesn't hurt.

File: sw/device/lib/crypto/data/crypto_testplan.hjson [hunk 1]
```diff
@@ -123,5 +125,17 @@
       bazel: ["//sw/device/tests/crypto:symmetric_keygen_functest"]
       lc_states: ["PROD"]
     }
+    {
+      name: crypto_lib_entropy
+      desc: '''Check the quality of entropy generated by PRNGs.
```
Comment (pamaury) [inline sw/device/lib/crypto/data/crypto_testplan.hjson:130]: I am not sure that a known-answer test is supposed to test the "quality" of the entropy, it is really more testing that the PRNG follows the specification

File: sw/device/lib/testing/test_framework/data/ottf_testplan.hjson [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+{
+  name: "ottf"
+  testpoints: [
+    {
+        name: ottf_crt
+        desc: '''Check that the crt for correctness.
+
+                 - Check that the .bss is zeroed.
+                 - Check that the .data (SRAM) was initialized with the .data_init (Flash)
+                 - Check that alined sections are zeroed.
```
Comment (pamaury) [inline sw/device/lib/testing/test_framework/data/ottf_testplan.hjson:13]: typo: "alined"?

File: sw/device/lib/testing/test_framework/data/ottf_testplan.hjson [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+                 - Check that data is correctly copyed across word aligned sections.
```
Comment (pamaury) [inline sw/device/lib/testing/test_framework/data/ottf_testplan.hjson:14]: copied?

**Review by pamaury — APPROVED** [review]

File: sw/device/lib/crypto/data/crypto_testplan.hjson (hunk 1 above)
Comment (engdoreis) [reply sw/device/lib/crypto/data/crypto_testplan.hjson]: Good point. I rephrased it.

**Pull request merged**